Test/collectiontest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+require_once('../functions.php');
+
+use PHPUnit\Framework\TestCase;
+
+class collectiontest extends TestCase {
+    public function testReturnsCharacterData() {
```

Comment:
what about a test where you pass it an array but it has the wrong keys in it? what will happen then?